PinpointKit/PinpointKit/Sources/FeedbackTableViewDataSource.swift

```diff
@@ -36,18 +40,56 @@ final class FeedbackTableViewDataSource: NSObject, UITableViewDataSource {
     }
     
     private enum Row {
+        case Screenshot(screensot: UIImage, hintText: String?, hintFont: UIFont)
```

Comment:
I believe this merely reflects the optionality of `InterfaceCustomization.Appearance.feedbackEditHintFont` and `InterfaceCustomization.Appearance.feedbackEditHint`. Really it is that we never won’t have a `hintFont` since it has a default value on initialization of `InterfaceCustomization`. How would you like to proceed? I can make `hintFont` optional here, but it will have no effect.